steamapi/store.py

```diff
@@ -0,0 +1,53 @@
+__author__ = 'andrew'
+
+import uuid
+from .core import APIConnection
+
+
+class SteamStore(object):
+    def __init__(self, appid, debug=False):
+        self.appid = appid
+        self.interface = 'ISteamMicroTxnSandbox' if debug else 'ISteamMicroTxn'
+
+    def get_user_microtxh_info(self, steamid):
+        params = {
+            'steamid': steamid,
+            'appid': self.appid
+        }
+        return APIConnection().call(self.interface, 'GetUserInfo', 'v1', **params)
```

Comment:
I prefer directly specifying keyword parameters if they're simple enough, like here. (Where you don't use any defaults)